src/js/components/Base.js

```diff
@@ -21,14 +28,15 @@ const Header = ({ currentUrl }) => {
         <//>
         <ul class="mb-6 space-x-6 sm:mt-6">
           ${links.map(
-            ({ href, title }) => html`
+            ({ href, title, external }) => html`
               <li class="inline">
                 <${Link}
                   class=${tw(
                     linkClasses,
                     currentUrl && currentUrl.startsWith(href) && 'font-bold'
                   )}
                   href=${href}
+                  target=${external ? '_blank' : '_self'}
```

Comment:
I've always learned that external links should be opened in new tab so didn't even think about it. Now that I thought about it, I still think that external links should be opened in a new tab 😁 For a regular user (like our mom) it can be confusing if they're browsing a website, click a link and suddenly they are on another website. Example:

1. Open kk.fi
2. Click 'Twitter'
3. Browse our Twitter account
4. See interesting retweet, go see someones profile etc, do some browsing inside Twitter
5. You're done with Twitter and close the tab, and suddenly you can't see the kk.fi tab anywhere??

Of course you and I know we can spam the back-button or go to kk.fi again, but that might be too big of a step for a not-so-tech-savvy person. Especially if they arrived on your website on accident and can't remember the url. Doesn't really matter in this case, but is a big deal if you want to keep the user engaging with your site as long as possible (more ad revenue oslt.)

That was basically a long way of saying that I think you should never close your own site on behalf of the user. However as a power-user I agree that it's irritating when a link opens in new tab when you don't want it to. And our website visitors will anyway be nerds so I agree that Twitter should not be opened in a new tab 😂

I'll make the change

Reply:
Agree about power users. 👍

I see your point about regular users. And I disagree. 😂 Alternative viewpoint: opening links in new tabs by default is how some users end up with dozens open tabs, hundreds even. I should ponder this more thoroughly by writing a blog post, but yea, we can anyway agree to disagree. 😀